pom.xml

```diff
@@ -198,6 +207,29 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>com.mycila</groupId>
+        <artifactId>license-maven-plugin</artifactId>
+        <version>4.3</version>
+        <configuration>
+          <properties>
+            <owner>${project.organization.name}</owner>
+            <email>info@ozone-his.com</email>
+            <year>2023</year>
+          </properties>
```

Comment:
These are the configurable properties. The year should be the inception year.
I don't have the correct information for this. @mks-d?

Reply:
@corneliouzbett where is the definition of `<year/>` documented, could you provide a link to it?

Reply:
The element `<year/>` is a license plugin placeholder/property as documented [here](https://oss.carbou.me/license-maven-plugin/), which should refer to `<inceptionYear/>` in the POM. So we define the project inception year in the POM.

What is the inception year if `eip-client` project?

Ref:
> inceptionYear: The year of the project's inception, specified with 4 digits. This value is used when generating copyright notices as well as being informational.

Reply:
> If this is supposed to track the inception of the overall Ozone project, that would be 2021.

This is per project(e.g `eip-client`, `eip-erpnext-openmrs`, ..., e.t.c).

Reply:
Then we have to look, for each project, from when dates the first commit...